jsonnet/rules.libsonnet

```diff
@@ -537,6 +537,14 @@ function(params) {
           expr: 'sum by (namespace) (max without(instance) (container_memory_working_set_bytes{namespace=~"openshift-monitoring|openshift-user-workload-monitoring", container=""}))',
           record: 'monitoring:container_memory_working_set_bytes:sum',
         },
+        {
+          expr: 'topk(10, sum without(instance)(sum_over_time(scrape_series_added[1h])))',
```

Comment:
I'm not so sure about the cost of returning 10 results, probably we can reduce to 5 if it is too costly.

Reply:
@ianbillett thoughts?

Reply:
There's no difference in terms of resources in returning 10 or 5 or 1 for that matter - the process will be gather -> sort -> return n. The amount of samples pulled into memory will be the same.

`openshift:scrape_series_added:sum` is not what the expression is returning tho? `openshift:scrape_series_added_top10:sum` would be more accuracte.

Reply:
We should consider a more accurate metric name as suggested by @ianbillett 

/label px-approved
/unassign @sferich888

Reply:
@ianbillett, what would be cost associated with telemetry on this metrics? I feel each cluster would add 20 additional series with this change.

>openshift:scrape_series_added:sum is not what the expression is returning tho? openshift:scrape_series_added_top10:sum would be more accuracte.

I'm open for this, however I feel this would cause label matching issues if we decide to go with different number in future. How about  `openshift:scrape_series_added:sum:top`?

Reply:
Returning 10 series is too expensive for telemetry. As a matter of fact, we are limited to ~10 series per team. 3 should already be good enough since we are interested in detecting the most expensive targets depending on the cluster size. But 5 should be also good depending on the budget we can allocate to this. @simonpasquier wdyt?

Reply:
I think that aggregating per target is too fine grained: it might generate new series whenever a new pod is rolled out (because its name would change for example). I suggest to do `topk(3, sum by (namespace,job) (sum_over_time(scrape_series_added[1h])))` which would provide a breakdown by namespace and service (same remark for `scrape_samples_added` obviously)

Reply:
Thanks @simonpasquier , @dgrisonnet .

>I think that aggregating per target is too fine grained: it might generate new series whenever a new pod is rolled out (because its name would change for example)

I agree with you to a certain extent., but the same applies for Service/Endpoint rename as well right?(`by(namespace,job)`)

Reply:
Addressed @simonpasquier 's comments, still the question on rule name is open :)